target/linux/octeon/base-files/lib/preinit/01_sysinfo

```diff
@@ -9,6 +9,10 @@ do_sysinfo_octeon() {
 		name="erlite"
 		;;
 
+	"UBNT_E120"*)
+		name="usg3p"
```

Comment:
Modern targets on OpenWrt use the compatible here, which is typically based on the device definition name, just with the underscore replaced by a comma:
ubnt,unifi-security-gateway-3p

I'd be happy if we could start that for new devices on this target as well.
This will affect names in /lib/upgrade/platform.sh, I will add a specific comment there.